lib/reporter/reporters/html/reporter.html.ejs

```diff
@@ -0,0 +1,129 @@
+    <link rel="stylesheet" href="<%= htmlFolder %>/css/style.css">
+    <link rel="stylesheet" href="<%= htmlFolder %>/fonts/museosans.css">
+  </head>
+  <body>
+
+    <div id="wrap">
+      <div id="main">
+        <section id="navigation">
+          <div class="container-fluid">
+            <nav class="navbar navbar-expand-lg">
+              <a class="navbar-brand" href="#"><div class="logo"></div></a>
+            </nav>
+          </div>
+        </section>
+        <section id="auxbar">
+          <div class="container-fluid">
+            <div class="row">
+              <div class="auxcol col-md-4 align-items-center">
+                <div class="auxcontainer">
+                  <div class="auxbox autobox">
+                  </div>
+                </div>
+              </div>
+              <div class="auxcol col-md-4 align-items-center">
+                <div class="auxcontainer align-h-center">
+                  <h1>Session Details - <strong>Test Case</strong></h1>
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+
+
+
+        <section id="sessions">
+          <div class="row">
+          <div class="col-md-12">
+          <div class="container-fluid">
+            <% Object.keys(modules).forEach(function(key, moduleIndex) {
+              var module = modules[key]; %>
+                <div id="s<%= moduleIndex+1 %>" class="sessionbox">
+                  <%- include('partials/test-module', {module: module, key: key, moduleIndex}) %>
+                <% }) %>
+          </div>
+          </div>
+         </div>
+        </section>
+      </div>
+    </div>
+
+    <section id="footer">
+      <div class="container-fluid">
+        <div class="row footerstarbg">
+          <div class="col-md-12">
+            <nav class="footer-box">
+              <div class="footer-links">
+                  <span class="cr" href="https://nightwatchjs.org">© 2022 Nighwatch.js</span>
+            </nav>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <!-- <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script> -->
+    <script src="<%= htmlFolder %>/js/jquery-3.2.1.min.js"></script>
+    <script src="<%= htmlFolder %>/js/popper.min.js"></script>
+    <script src="<%= htmlFolder %>/js/bootstrap.min.js"></script>
```

Comment:
is this needed?

Reply:
Yes for opening and closing of the dialogue box with assertion details.